File: src/main/java/com/server/ordering/domain/dto/response/RecentOrderRestaurantDto.java
package com.server.ordering.domain.dto.response;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import static lombok.AccessLevel.PROTECTED;

@Getter @Setter
@AllArgsConstructor
@NoArgsConstructor(access = PROTECTED)
public class RecentOrderRestaurantDto {

    private Long restaurantId;
    private String restaurantName;
    private String profileImageUrl;
    private String backgroundImageUrl;
    private float rating;
    private int orderingWaitingTime;
}

File: src/main/kotlin/com/byeoru/ordering_server/domain/dto/response/RecentOrderRestaurantDto.kt
package com.byeoru.ordering_server.domain.dto.response

data class RecentOrderRestaurantDto(val restaurantId: Long,
                                    val restaurantName: String,
                                    val profileImageUrl: String,
                                    val backgroundImageUrl: String,
                                    val rating: Float,
                                    val orderingWaitingTime: Int)